ui/app/components/oidc-auth-buttons/index.hbs

```diff
@@ -5,7 +5,7 @@
   data-test-oidc-provider={{authMethod.name}}
   {{on "click" (perform this.initializeOIDCFlow authMethod)}}>
     {{#if this.initializeOIDCFlow.isRunning}}
-      <Spinner @size="16"/>
+      <FlightIcon @name="loading" @size="16" />
```

Comment:
I wasn't sure how to get QA how the icon looks in OIDC in Mirage, ditto to `project-init-running`. Any ideas? Only have 16 and 24 option in Flight.

Reply:
Here’s how it looks on this branch:

![localhost_4200_auth](https://user-images.githubusercontent.com/34030/199709825-211ae6c6-5fb4-468a-bf32-a8abaad9cf0b.png)

And here’s how it looks on `main` (seems to be broken!):

![localhost_4200_auth (1)](https://user-images.githubusercontent.com/34030/199709970-0672e409-55e4-4446-ac97-eb103b2c6d9d.png)

I fudged the templates to get into this state.

<details>
<summary>🤠 <b>Patch</b></summary>

```patch
diff --git a/ui/app/components/oidc-auth-buttons/index.hbs b/ui/app/components/oidc-auth-buttons/index.hbs
index 636ebdf47..c4b5cc016 100644
--- a/ui/app/components/oidc-auth-buttons/index.hbs
+++ b/ui/app/components/oidc-auth-buttons/index.hbs
@@ -1,10 +1,10 @@
-{{#each @model.authMethodsList as |authMethod|}}
+{{#each (array (hash name="Initech SSO" displayName="Initech SSO")) as |authMethod|}}
   <Pds::Button
   @variant="primary"
   disabled={{this.initializeOIDCFlow.isRunning}}
   data-test-oidc-provider={{authMethod.name}}
   {{on "click" (perform this.initializeOIDCFlow authMethod)}}>
-    {{#if this.initializeOIDCFlow.isRunning}}
+    {{#if true}}
       <Spinner @size="16"/>
     {{/if}}
     {{authMethod.displayName}}
diff --git a/ui/app/templates/auth/index.hbs b/ui/app/templates/auth/index.hbs
index 42ab1e527..36bf8c8fe 100644
--- a/ui/app/templates/auth/index.hbs
+++ b/ui/app/templates/auth/index.hbs
@@ -1,6 +1,6 @@
 <h1>{{t 'auth.title'}}</h1>
 <p>{{t 'auth.subtitle'}}</p>
-{{#if @model.authMethodsList}}
+{{#if true}}
   <OidcAuthButtons @model={{@model}}/>
 {{else}}
   <p>

```
</details>